pages/dkp/kaptain/2.1.0/user-management/active-directory/index.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: layout.pug
+navigationTitle: Active Directory
+title: Authenticate users using Active Directory
+menuWeight: 30
+excerpt: Authenticate users to access Kaptain with Active Directory (AD) groups.
+beta: false
+enterprise: false
+---
+
+Authenticate users to access Kaptain with Microsoft’s Active Directory (AD) groups.
+
+Kaptain relies on Dex as its identity provider, a service that enables the management of users and user groups. Dex, in turn, delegates to one or more external identity providers like AD.
+
+The Active Directory (AD) feature allows you to manage and control access to your Kaptain instance, by using an existing rule set of permissions established in the AD (via the AD admin interface) to authenticate and grant access to Kaptain users and user groups.
+
+You can activate AD to authenticate users regardless of your environment set up (air-gapped or networked), or infrastructure provider.
+
+If you are already using AD groups to authenticate users in DKP or other platforms, this feature will allow you to have centralized access management and control. It will also allow your users to access Kaptains's Kubeflow dashboard with the same credentials they have been using for other platforms.
+
+When the AD is active, users will be able to log in to Kaptain with their AD credentials. For this, Kaptain’s dex uses LDAP to call up and verify the entered credentials against your existing AD groups.
```

Comment:
I am confused how any of this is 'Active Directory' specific.   Active Directory implements the LDAP protocol so any client that can talk LDAP can talk to AD, and that's how DEX connects to AD.  

It seems like what we've added in this new feature in Kaptain is the ability to limit what groups in LDAP have access to Kaptain.    So I believe this capability would work with any LDAP Directory, not just Active Directory...